### PR TITLE
fix(prefixes): warn about unrecognized prefix instead of error

### PR DESCRIPTION
**Pull request by noelmcloughlin**
This PR allows documentation to be built despite https://github.com/linkml/linkml-runtime/issues/96

But a solution to #96 would be better of course..

### CODE REVIEW

**Comment by noelmcloughlin** [comment]
image

File: linkml_runtime/utils/namespaces.py [hunk 1]
```diff
@@ -192,7 +192,9 @@ def uri_or_curie_for(self, prefix: Union[str, URIRef], suffix: str) -> str:
                     return k + ':' + suffix
             return self.join(str(prefix), suffix)
         elif prefix not in self:
-            raise ValueError(f"Unrecognized prefix: {prefix}")
+            # raise ValueError(f"Unrecognized prefix: {prefix}")
+            print(f"WARNING: Unrecognized prefix: {prefix}")
```
Comment (cmungall) [inline linkml_runtime/utils/namespaces.py:196]: I would prefer logging.error or something that writes on stderr

**Review by cmungall — CHANGES_REQUESTED** [review]

File: linkml_runtime/utils/namespaces.py (hunk 1 above)
Comment (noelmcloughlin) [reply linkml_runtime/utils/namespaces.py:196]: Yes, this is not ideal solution - I'm happy to close or change the PR while I investigate #96 further to be honest.

**Comment by noelmcloughlin** [comment]
Closing see https://github.com/linkml/linkml-runtime/issues/96

**Pull request closed without merging**